[2023-01-04T19:13:53.856-0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: get_epitopes.vdjdb manual__2023-01-04T22:13:31.509436+00:00 [queued]>
[2023-01-04T19:13:53.866-0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: get_epitopes.vdjdb manual__2023-01-04T22:13:31.509436+00:00 [queued]>
[2023-01-04T19:13:53.866-0300] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T19:13:53.866-0300] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-01-04T19:13:53.866-0300] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-04T19:13:53.877-0300] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): vdjdb> on 2023-01-04 22:13:31.509436+00:00
[2023-01-04T19:13:53.880-0300] {standard_task_runner.py:55} INFO - Started process 7444 to run task
[2023-01-04T19:13:53.882-0300] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_epitopes', 'vdjdb', 'manual__2023-01-04T22:13:31.509436+00:00', '--job-id', '246', '--raw', '--subdir', 'DAGS_FOLDER/get_epitopes.py', '--cfg-path', '/tmp/tmpfoj4d_5l']
[2023-01-04T19:13:53.883-0300] {standard_task_runner.py:83} INFO - Job 246: Subtask vdjdb
[2023-01-04T19:13:53.945-0300] {task_command.py:389} INFO - Running <TaskInstance: get_epitopes.vdjdb manual__2023-01-04T22:13:31.509436+00:00 [running]> on host kallo-Ubuntu
[2023-01-04T19:13:54.014-0300] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Kallox
AIRFLOW_CTX_DAG_ID=get_epitopes
AIRFLOW_CTX_TASK_ID=vdjdb
AIRFLOW_CTX_EXECUTION_DATE=2023-01-04T22:13:31.509436+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-04T22:13:31.509436+00:00
[2023-01-04T19:13:54.603-0300] {python.py:177} INFO - Done. Returned value was: None
[2023-01-04T19:13:54.614-0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=get_epitopes, task_id=vdjdb, execution_date=20230104T221331, start_date=20230104T221353, end_date=20230104T221354
[2023-01-04T19:13:54.655-0300] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-04T19:13:54.679-0300] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
